modules/op-using-serviceaccounts.adoc

```diff
@@ -0,0 +1,91 @@
+// This module is included in the following assembly:
+//
+// *openshift-docs/cicd/pipelines/authenticating-pipelines-and-tasks-using-secrets.adoc
+
+[id="op-using-serviceaccounts_{context}"]
+= Using ServiceAccounts
+
+This approach involves the following steps:
+
+. Annotating specific secrets using either the Git or Docker annotation
+
+. Linking the annotated `Secrets` to `ServiceAccounts`
+
+. Specifying the `ServiceAccounts` on PipelineRun or TaskRuns
+
+== Annotations
+
+Tekton offers two types of authentication for secrets: 
+
+* `tekton.dev/git-{}` - Used to mark the annotated secret as a kid secret. 
+
+* `tekton.dev/docker-{}` - Used to mark the annotated secret as a docker secret.
+
+Additionally, Tekton supports different types of secrets for each authentication type. 
+
+For `git` it supports the following types of secrets:
+
+* kubernetes.io/basic-auth : basic authentication 
+
+* kubernetes.io/ssh-auth : ssh based authentication (keys, …)
+
+
+For `docker` it supports the following types of secrets:
+
+* kubernetes.io/basic-auth : basic authentications 
+
+* kubernetes.io/dockercfg : serialized ~/.dockercfg file kubernetes.io/dockerconfigjson : serialized ~/.docker/config.json file
+
+
+== Linking the annotated secrets to  ServiceAccounts
+
+To link a secret to a _ServiceAccount_, you can reference the Secret by the name in the *secrets* field of the _ServiceAccount_. For example:
+
+[source,yaml,subs="attributes+"]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: build-bot
+secrets:
+  - name: regcred
+  - name: a-git-auth-secret
+----
+
+== Specifying the ServiceAccounts on PipelineRun or TaskRuns
+
+To specify a _ServiceAccount_ for a _PipelineRun_ or a _TaskRun_, use the `serviceAccountName` field,
+
+[source,yaml,subs="attributes+"]
+----
+# For a TaskRun
+
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: build-with-basic-auth
+spec:
+  serviceAccountName: build-bot
```

Comment:
v1 API uses different syntax https://tekton.dev/docs/pipelines/migrating-v1beta1-to-v1/#changes-to-fields